src/index.js

```diff
@@ -120,7 +120,8 @@ export type ResizableProps = {
   onResize?: ResizeCallback,
   onResizeStop?: ResizeCallback,
   defaultSize?: Size,
-  scale?: number
+  scale?: number,
+  movementRatio?: number,
```

Comment:
@martinmcneela Sorry for late. I think it is better to rename it to `resizeRatio`.
And could you please add this new props to `README` ?